api/src/main/java/org/openmrs/module/locationbasedaccess/utils/LocationUtils.java

```diff
@@ -36,47 +38,52 @@ public static Location getPersonLocation(Person person) {
         return null;
     }
 
-    public static Boolean doesPersonBelongToGivenLocation(Person person, PersonAttributeType personAttributeType, String sessionLocationUuid) {
+    public static Boolean doesPersonBelongToGivenLocations(Person person, PersonAttributeType personAttributeType, List<String> sessionLocationsLists) {
```

Comment:
What is the use case of this change and why we need it? please clarify for me

Reply:
This method is used in person and patient serviceIntrepreterAdvice's to check whether they belong to current user Locations List.